### ISSUE TITLE
Edit optional information

### ISSUE THREAD

**Issue by AndreaLeto00** [grokability/snipe-it#11473]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

As per the title, after the latest updates, an item that I consider very important when filling out the form to create a new asset has ended up among the optional information, as pictured:
![image](https://user-images.githubusercontent.com/78138698/177730400-9c6fa2fc-9342-45ef-bf00-dff863e9084f.png)

is there any way to move it to the top?

### Reproduction steps

1.
2.
3.
...

### Expected behavior

As per the title, after the latest updates, an item that I consider very important when filling out the form to create a new asset has ended up among the optional information, as pictured:

is there any way to move it to the top?

### Snipe-IT Version

Version v6.0.7 - build 8351 (master) 

### Operating System

Ubuntu

### Web Server

Apache

### PHP Version

7.4+

**Comment by snipe** [comment]
The disclosure arrow should be remembering whether it was opened or closed the last time you opened or closed it. If you need those fields, just leave the disclosure arrow open.

**Comment by Tekk3r** [comment]
I don't agree with this change either. The asset name is one of our main fields we reference. Not exactly operational friendly to have it at the very bottom of the screen now.

**Comment by ATPTech** [comment]
We ran into this as well. Is there a possibility of at least having an option to making this "optional" or "required" like we do our custom fields?

**Comment by snipe** [comment]
@ATPTech that's not on the roadmap at this time. We literally didn't change anything other than shifting fields around. Nothing is required that wasn't before, and nothing is optional that wasn't before. 

If the ask is just "move asset name back out of the disclosure arrows" someone could have just said that. If the ask is "make asset name required", someone could have just said that too. 

The arrows are remembered, so once you submit the form, the state of the arrows is remembered. 

People complained that the form was too long. We listened, and made it smaller. Now people complain that it's too short - even though it's exactly the same size, just tucking lesser used options away.

**Comment by ATPTech** [comment]
@snipe 

I think in this case, the ask is more **we think having an optional disclosure field is awesome**, if we can choose what fields are considered "optional", just like we can for custom fields. 

"Asset Name" for us, is considered mandatory. But it's being classified as "optional". 

![image](https://user-images.githubusercontent.com/111889165/186205379-771e1bb7-ef0d-484e-9766-108a817794c6.png)

It's not the end of the world. We'll train our team on the change. 

I think you implementing optional fields is awesome. If we were able to choose which fields are considered optional, that would be our "ask".  I can create a new ticket. I was just googling around and found this recent thread.

**Comment by snipe** [comment]
Those fields were always optional though, that's what's so confusing to me about some folks' reactions to this change.

**Comment by snipe** [comment]
There are lots of things that cannot be considered optional. The system depends on them having values, and our entire validation system bakes those things in. It seems the better option is to remove "asset name" and let people make that a custom field that they can then make required.

**Comment by mihaiolimpiu** [comment]
**I very much agree with @ATPTech's view on this, I see both of your views clashing, and I'll try to explain better,**

**Short story:**
Mandatory classification, Custom Ordering, and custom categories would fit the bill for the Add or edit pages :)
Or if we would be more realistic:
Name  is far from optional. (Invoice item name - Description if you want)
Notes, Requestable, Add Image, Warranty are way better suited to be optional.

**Long Story:**
I'll try to explain what the issue really is.
As I see it Snipe it tries to push more granular models and models names, but in practice it's hard to manage things like that when you manage 10k + assets as it forces users to create new models for every little model change.

As an example:
Manufacturer has the model name VentureX, Model Number 5500
In a couple of months they release Model Number 5501, 5510, 5511 with their variations A,B,C,D.

You are required to have at least 16 Venture X models:
Venture X 5500, A, B, C, D
Venture X 5501, A, B, C, D
Venture X 5510, A, B, C, D
Venture X 5511, A, B, C, D

Well, Most of us like to keep it simple so we only have a single VentureX model, and in the asset name we write: Venture X 5501 A as that is usually on the invoice.
**Advantage, on the model picker instead of choosing from minimum 16 VentureX models you are looking only for that single one.**

I hope I make sense.

**Comment by repdk** [comment]
I would also like to see this changed.
In my organization, the Asset name field is considered as an important field to fill out.

Please revert this change, or at least give us the option to decide what is "optional".

**Comment by MouseMadeContent** [comment]
Hey, we had the same requirement for the asset-name tag.
Here is what i have done:

Edit: resources/views/partials/forms/edit/name.blade.php
 
replace:

`<input class="form-control" type="text" name="name" aria-label="name" id="name" value="{{ old('name', $item->name) }}"{!!  (Helper::checkIfRequired($item, 'name')) ? ' data-validation="required" required' : '' !!} />`

with:

` <input class="form-control" type="text" name="name" aria-label="name" id="name" value="{{ old('name', $item->name) }}"  data-validation="required">`


Edit: resources/views/hardware/edit.blade.php

Add: Line 16: @include ('partials.forms.edit.name', ['translated_name' => trans('admin/hardware/form.name'),  'field_req' => true])
Remove: Line 121:  @include ('partials.forms.edit.name', ['translated_name' => trans('admin/hardware/form.name')])

That puts the asset-name field on the top with the "require"-tag.
I didn't test it on a long term period, so please make backups of the 2 Files before editing :)

![image](https://user-images.githubusercontent.com/34419201/211858147-423b8161-37a5-4f73-9b32-1307a6f391cc.png)

Br
Chris
